Sources/Colors/ColorProvider.swift

```diff
@@ -1172,33 +1274,39 @@ public struct UIColorProvider {
     var pillFilterIcon: UIColor { token.iconInverted }
     var pillSuggestionText: UIColor { token.text }
     var pillSuggestionBackground: UIColor {
-        switch Warp.Config.warpTheme {
+        switch Warp.Theme {
         case .finn:
             return UIColor.dynamicColor(defaultColor: FinnUIColors.gray200, darkModeColor: FinnUIColors.gray600)
         case .tori:
             return UIColor.dynamicColor(defaultColor: ToriUIColors.gray200, darkModeColor: ToriUIColors.gray600)
         case .dba:
             return UIColor.dynamicColor(defaultColor: DBAUIColors.gray200, darkModeColor: DBAUIColors.gray600)
+        case .blocket:
+            return UIColor.dynamicColor(defaultColor: BlocketUIColors.gray200, darkModeColor: BlocketUIColors.gray600)
         }
     }
     var pillSuggestionBackgroundHover: UIColor {
-        switch Warp.Config.warpTheme {
+        switch Warp.Theme {
         case .finn:
             return UIColor.dynamicColor(defaultColor: FinnUIColors.gray300, darkModeColor: FinnUIColors.gray500)
         case .tori:
             return UIColor.dynamicColor(defaultColor: ToriUIColors.gray300, darkModeColor: ToriUIColors.gray500)
         case .dba:
             return UIColor.dynamicColor(defaultColor: DBAUIColors.gray300, darkModeColor: DBAUIColors.gray500)
+        case .blocket:
+            return UIColor.dynamicColor(defaultColor: BlocketUIColors.gray300, darkModeColor: BlocketUIColors.gray500)
         }
     }
     var pillSuggestionBackgroundActive: UIColor {
-        switch Warp.Config.warpTheme {
+        switch Warp.Theme {
         case .finn:
             return FinnUIColors.gray400
         case .tori:
             return ToriUIColors.gray400
         case .dba:
             return DBAUIColors.gray400
+        case .blocket:
+            return DBAUIColors.gray400
```

Comment:
Should be `BlocketUIColors`
```suggestion
            return BlocketUIColors.gray400
```